{"name": "test_required_login_password", "status": "broken", "statusDetails": {"message": "AttributeError: 'Response' object has no attribute 'get'", "trace": "self = <tests.test_login_user.TestLoginUser object at 0x000001DE5E1FBE90>\n\n    @allure.description('Получение статус-кода с неверным логином и паролем')\n    def test_required_login_password(self):\n        response = requests.post(urls.CREATE_USER, data=data.payload)\n>       email = response.get('email', '')\nE       AttributeError: 'Response' object has no attribute 'get'\n\ntests\\test_login_user.py:24: AttributeError"}, "description": "Получение статус-кода с неверным логином и паролем", "start": 1725651247733, "stop": 1725651247967, "uuid": "0e554a84-63fc-4444-a17f-481b11adf7c5", "historyId": "c19965dcb797bd33784c88cc1844c096", "testCaseId": "c19965dcb797bd33784c88cc1844c096", "fullName": "tests.test_login_user.TestLoginUser#test_required_login_password", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "DESKTOP-ODIODOM"}, {"name": "thread", "value": "14732-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_user"}]}